tests/unit/test_app_endpoints.py

```diff
@@ -0,0 +1,122 @@
+"""Unit tests for app endpoints."""
+
+import os
+from http import HTTPStatus
+from unittest.mock import patch
+
+import pytest
+from fastapi import HTTPException
+
+from ols import constants
+from ols.app.endpoints import feedback, ols
+from ols.app.models.models import FeedbackRequest, LLMRequest
+from ols.utils import config
+
+
+@pytest.fixture(scope="module")
+@patch.dict(os.environ, {"LOG_LEVEL": "INFO"})
+@patch.dict(os.environ, {"BAM_API_KEY": "BAM API KEY"})
```

Comment:
more stuff that's going to be impacted by the switch to config.yaml loading instead of env loading.

cc @syedriko

Reply:
when we remove the function `config.load_config_from_env()`, we can update this fixture function. The only purpose is to instantiate the logger which is called in many places in the codes to test. 
Using a mock on `LLMLoader` and `LLMChain` the variable `BAM_API_KEY`  becomes useless and will be removed.